atomic_reactor/plugins/exit_sendmail.py

```diff
@@ -43,7 +43,7 @@
 # RFC 2821 defines a valid email address much more expansively, but this definition matches
 # what most people loosely expect to be valid.
 # specifically, email addresses of the form id/name@domain.tla are not valid.
-VALID_EMAIL_REGEX = '^[a-zA-Z][a-zA-Z0-9-_\.\+]+@[a-zA-Z0-9-\.]+\.[a-zA-Z0-9-]+$'
```

Comment:
Using raw string for regexp is ok. However escaping `.` and `+` inside `[ ]` is not needed and can be removed in favor of readability